src/components/GiftList/GiftList.js

```diff
@@ -15,17 +15,21 @@ const GiftList = ({ id }) => {
     console.log(gifts)
   },[giftMockData.data, id]);
 
-  return (
-    // const giftCards = gifts.map(gift => {
-    //   return (
-
-    //   )
-    // })
+    const giftCards = gifts.map(gift => {
+      return (
```

Comment:
Future us problem but when we get the gift api we will need to make a function so when a user adds a gift it will add to the existing list.